drivers/pinmux/stm32/pinmux_stm32f4.h

```diff
@@ -451,6 +451,15 @@
 #define STM32F4_PINMUX_FUNC_PE13_PWM1_CH3    \
 	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUSHPULL_PULLUP)
 
+#define STM32F4_PINMUX_FUNC_PE11_SPI4_NSS   \
+	(STM32_PINMUX_ALT_FUNC_5 | STM32_PUSHPULL_NOPULL)
+#define STM32F4_PINMUX_FUNC_PE12_SPI4_SCK \
+	(STM32_PINMUX_ALT_FUNC_5 | STM32_PUSHPULL_NOPULL)
+#define STM32F4_PINMUX_FUNC_PE13_SPI4_MISO \
+	(STM32_PINMUX_ALT_FUNC_5 | STM32_PUSHPULL_NOPULL)
+#define STM32F4_PINMUX_FUNC_PE14_SPI4_MOSI \
+	(STM32_PINMUX_ALT_FUNC_5 | STM32_PUSHPULL_NOPULL)
+
```

Comment:
Can you put reorder the lines to accommodate with the global file structure?

Reply:
Fixed.